subcommittees/local-group/local_regional_mentor_communities.md

```diff
@@ -0,0 +1,46 @@
+### Prospectus for Local/Regional Mentor/Community groups
```

Comment:
I can do that, the I cam into the process a bit backdoor and am coming catching up on the process.